dotnet/sample-apps/aws-sdk/wrapper/SampleApps/AwsSdkSample/obj/Debug/net6.0/.NETCoreApp,Version=v6.0.AssemblyAttributes.cs

```diff
@@ -0,0 +1,4 @@
+// <autogenerated />
```

Comment:
I think the changes to the `dotnet` folder need to be removed.

Reply:
Ok. I'll do that

Reply:
Done @tylerbenson any further changes?

Reply:
I meant the whole `dotnet` folder... not just that one file.  I don't know why those are checked in.

Reply:
Yes, I did notice that those files somewhat gets automatically added after cloning the repo and running a build, hence why they are being commited as changes, not sure why though.

Reply:
Done now, you should be seeing only the changes in the `.github/workflows` now @tylerbenson